pkgs/development/tools/kind/default.nix

```
@@ -4,18 +4,19 @@ with stdenv.lib;
 
 buildGoPackage rec {
   name = "kind-${version}";
-  version = "0.2.1";
+  version = "v0.3.0";
```

According to the manual, the version should start by a digit.
```suggestion
  version = "0.3.0";
```